### ISSUE TITLE
Ability to choose which version of Python to use

### ISSUE THREAD

**Issue by bjkomer** [nengo/nengo_gui#825]
I would be nice if there was an easy to specify/switch which Python version is used, via a command line argument when running `nengo_gui` and/or a configuration option.

**Comment by jgosmann** [comment]
I don't think this is possible because `nengo_gui` will start the Python interpreter before we can parse the command line arguments. Except maybe if we do something like Jupyter with it's kernels, but that seems to be somewhat out of the scope of the GUI.

The proper way to handle this is to install `nengo_gui` for each Python version with the respective pip (e.g. `pip2 install nengo_gui` and `pip3 install nengo_gui`) or run `setup.py` with the respective Python interpreter. You might need not provide the full path to one of the nengo_gui though, but you could create an alias for that.

**Comment by tbekolay** [comment]
You can see what Python will be used in the current environment by doing:

```bash
cat $(which nengo)
```

The top line is the [shebang](https://en.wikipedia.org/wiki/Shebang_(Unix)) that defines what executable is used to run the `nengo` script (which is generated when you do `pip install` or `setup.py install/develop`).

If we wanted to change it we'd essentially have to have the main script not actually start the Nengo server, but instead spawn a Python subprocess that starts the Nengo server. Or, we spawn a separate Python process to do the interaction with the Nengo model (or maybe we already do that?)
